MobiFlight/ExecutionManager.cs

```diff
@@ -787,7 +787,7 @@ private string ExecuteDisplay(string value, OutputConfigItem cfg)
                         {
                             string refValue = FindValueForRef(cfg.LedModule.DisplayLedBrightnessReference);
 
-                            mobiFlightCache.setDisplayBrightness(
+                            mobiFlightCache.SetDisplayBrightness(

```

Comment:
This kind of change (and subsequent) are not warnings. These are messages.

Reply:
Yeah, that's what I meant. They're code style warnings but report as messages in the errors list.